library/core/src/num/f16.rs

```diff
@@ -344,6 +353,160 @@ impl f16 {
         self.abs_private() < Self::INFINITY
     }
 
+    /// Returns `true` if the number is [subnormal].
+    ///
+    /// ```
+    /// #![feature(f16)]
+    /// # #[cfg(target_arch = "aarch64")] { // FIXME(f16_F128): rust-lang/rust#123885
```

Comment:
Should this also be `#![cfg]`?

Should it be `cfg(not)`?

Reply:
Other question: Should this be `cfg(reliable_f16)`?

Reply:
That is only in `std`, I didn't duplicate the logic to core so it doesn't live in two places. Wish there was a bit of an easier way, but a cargo feature or loading the info in a separate file seemed like overkill when these get more thoroughly verified in `std`'s tests. So running these doctests on any platform is really just a sanity check that they aren't _totally_ broken.

Reply:
That seems fine then!